src/SFA.DAS.ApprenticeCommitments.Api/nlog.config

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:xdt="http://schemas.microsoft.com/XML-Document-Transform">
+    <extensions>
+        <add assembly="SFA.DAS.NLog.Targets.Redis" />
+    </extensions>
+    <variable name="appName" value="das-employer-incentives-outer-api"/>
```

Comment:
this should be 'das-apprentice-commitments-outer-api'

Reply:
Updated